fiftyone/operators/executor.py

```
@@ -783,42 +791,66 @@ def target_view(self, param_name="view_target", require_flat=False):
-                flattened to the current active slice.
+                flattened to the current active slice
 
         Returns:
             a :class:`fiftyone.core.collections.SampleCollection`
         """
-        target = self.params.get(param_name)
+        return self._resolve_target_view(
+            self.params.get(param_name), require_flat=require_flat
+        )
 
+    def _get_target_collection(self, target):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_get_target_collection`

(ANN202)